status-im/status-react
# [#5213] allow user to select fleet in settings

fixes #5213 
(based off another PR for 5527)

### Summary:

- get wnodes from resources/config/fleets.json which is taken from fleets.status.im
- store wnodes by fleet and not by network since they are always the same
- reset wnodes settings during migration
- add option in developper menu to select fleet
- mailservers are now presented by their real name

### Steps to test:
- Open Status
- Check that fleet can be changed
- Check that mailserver settings are consistent when changing fleet (should randomly select a new mailserver)
(use cases list is wip)
- Check that go loglevel can be changed as well since the PR was merged into this one as well

status: ready
